src/modules/students/actions/index.js

```diff
@@ -27,7 +27,7 @@ export function joinClassroom(id, token) {
       data: json.data,
       members: json.included
     }))
-    .then(() => browserHistory.push('/students/classrooms/'))
+    .then(() => browserHistory.push('/students/'))
     .catch(response => console.log('RESPONSE-error: ', response))
```

Comment:
I want to recommend changing this to `.then(() => browserHistory.push('/students/assignments/'))` so students immediately see the Classroom they joined... but this may not work if the Classroom they joined doesn't have any Assignments the student has been assigned to. Which they wouldn't be, because they haven't joined the class at the time.